### ISSUE TITLE
Contract type campaign wrongly displayed if logging->trackerImpressions is false

### ISSUE THREAD

**Issue by somsak** [revive-adserver/revive-adserver#683]
When logging->trackerImpressions configuration is set to false (or disabled via "Conversion tracking settings -> Enable conversion tracking"). The displayed in campaign edit (campaign-edit.php) is oddly displayed.
1. If the "Set specific date" is set and we try to edit the campaign, the calendar in the right side will not be displayed. We have to choose something on the web first (such as selecting "Start immediately") and select "Set specific date" back again to make the calendar appear.
2. "Impression limit per day" in "Priority in relation to other campaigns" is also missing. Have to select any priority other than the one being selected to make this appear.

We traced back the code and found that the "conversions" element in this page will not appear if trackerImpressions is not enabled, thus the javascript formFormat in initCampaign will throw an exception that prevent the calendar and impression per day to appear.

**Comment by erikgeurts** [comment]
This sounds like an issue I've seen in the past, but as far as I know it's been resolved some time ago.

Can you please check and confirm you're running the latest version of Revive Adserver (v3.2.x+)?

**Comment by somsak** [comment]
```
[root@reviveads reviveads]# grep VERSION constants.php
    define('VERSION', '3.2.2');
```

We are using 3.2.2. This is rather fresh installed.

Note that the system is a migration from OpenX 2.8.11. 

I checked in www/admin/assets/js/ox.ui.js, I think you meant this snippet
```
    if ($conversionsField) {
        formFormat($conversionsField.get(0), true);
    }
```
It seems that conversionsField exists, but it's an empty array.

**Comment by mbeccati** [comment]
@erikgeurts I'm not sure it has been resolved. I believe we might just have suggested to re-enable conversions. Or maybe I have a patch lying around somewhere?

**Comment by erikgeurts** [comment]
@mbeccati I seem to recall @andrewatfornax worked on this in the past, I have searched around a bit, but I can't find it.

**Comment by andrewatfornax** [comment]
@erikgeurts, you may be thinking of #808?

**Comment by andrewatfornax** [comment]
@somsak - Please try an upgrade to the latest version, or apply the patch in #808. I will close this ticket for now, on the assumption that the issue is already resolved, but please let us know if it's not resolved by #808. Thanks.
